2024-03-06 09:38:43,001 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-06 09:38:43,001 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Configure stats pid to 17592
2024-03-06 09:38:43,001 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-06 09:38:43,002 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-06 09:38:43,002 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-06 09:38:43,002 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_093842-6gs7ebn1\logs\debug.log
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_093842-6gs7ebn1\logs\debug-internal.log
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000021DBA639310>
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:init():566] calling init triggers
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:init():616] starting backend
2024-03-06 09:38:43,011 INFO    MainThread:17592 [wandb_init.py:init():620] setting up manager
2024-03-06 09:38:43,014 INFO    MainThread:17592 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-06 09:38:43,016 INFO    MainThread:17592 [wandb_init.py:init():628] backend started and connected
2024-03-06 09:38:43,025 INFO    MainThread:17592 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-06 09:38:43,026 INFO    MainThread:17592 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-06 09:38:43,026 INFO    MainThread:17592 [wandb_init.py:init():720] updated telemetry
2024-03-06 09:38:43,037 INFO    MainThread:17592 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-06 09:38:43,993 INFO    MainThread:17592 [wandb_run.py:_on_init():2262] communicating current version
2024-03-06 09:38:44,297 INFO    MainThread:17592 [wandb_run.py:_on_init():2271] got version response 
2024-03-06 09:38:44,297 INFO    MainThread:17592 [wandb_init.py:init():804] starting run threads in backend
2024-03-06 09:38:55,491 INFO    MainThread:17592 [wandb_run.py:_console_start():2241] atexit reg
2024-03-06 09:38:55,491 INFO    MainThread:17592 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-06 09:38:55,491 INFO    MainThread:17592 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-06 09:38:55,491 INFO    MainThread:17592 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-06 09:38:55,492 INFO    MainThread:17592 [wandb_init.py:init():847] run started, returning control to user process
2024-03-06 09:39:25,114 INFO    MainThread:17592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-06 09:39:25,114 INFO    MainThread:17592 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-06 09:43:21,146 INFO    MainThread:17592 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-06 09:43:21,148 INFO    MainThread:17592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-06 09:43:21,148 INFO    MainThread:17592 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-06 09:44:13,554 INFO    MainThread:17592 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-06 09:44:13,557 INFO    MainThread:17592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-06 09:44:13,557 INFO    MainThread:17592 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-06 09:46:13,387 INFO    MainThread:17592 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Configure stats pid to 17592
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-06 09:46:13,401 INFO    MainThread:17592 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-03-06 09:46:13,402 INFO    MainThread:17592 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\logs\debug.log
2024-03-06 09:46:13,402 INFO    MainThread:17592 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240306_094613-oyjusw69\logs\debug-internal.log
2024-03-06 09:46:13,402 INFO    MainThread:17592 [wandb_init.py:init():566] calling init triggers
2024-03-06 09:46:13,402 INFO    MainThread:17592 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-06 09:46:13,402 INFO    MainThread:17592 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 6gs7ebn1
2024-03-06 09:46:13,405 INFO    MainThread:17592 [wandb_run.py:_finish():1970] finishing run tejas591995/mnist-training/6gs7ebn1
2024-03-06 09:46:13,405 INFO    MainThread:17592 [jupyter.py:save_history():445] not saving jupyter history
2024-03-06 09:46:13,406 INFO    MainThread:17592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-06 09:46:13,406 INFO    MainThread:17592 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-06 09:46:13,406 INFO    MainThread:17592 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-06 09:46:13,407 INFO    MainThread:17592 [wandb_run.py:_restore():2193] restore
2024-03-06 09:46:13,407 INFO    MainThread:17592 [wandb_run.py:_restore():2199] restore done
2024-03-06 09:46:19,532 INFO    MainThread:17592 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-06 09:46:19,532 INFO    MainThread:17592 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-06 09:46:19,537 INFO    MainThread:17592 [wandb_run.py:_footer_sync_info():3810] logging synced files
